docs/relationships.md

```diff
@@ -42,59 +52,50 @@ const client = new CozyClient({
 })
 ```
 
-With the help of the schema, Cozy-Client will be able to manage the `relationships` attribute of your documents for you.
+We explain below what are the different types of relations.
```

Comment:
Sounds a bit redundant, isn't the `### Relation Types` title enough here?

Reply:
Personally, when I encounter a new concept, such as the relation type here, I like to be informed right ahead that this concept will be explained later, without having to skip to the next section :)